code/datums/quirks/negative_quirks/negative_quirks.dm

```diff
@@ -445,6 +447,74 @@
 		quirk_holder.toggle_move_intent()
 	quirk_holder.add_mood_event("nyctophobia", /datum/mood_event/nyctophobia)
 
+/datum/quirk/photophobia
+	name = "Photophobia"
+	desc = "Bright lights irritate you. Your eyes start to water and burn when exposed to light. Maybe it's a medical condition."
+	icon = FA_ICON_ARROWS_TO_EYE
+	value = -4
+	gain_text = span_danger("The safety of light feels off...")
+	lose_text = span_notice("Enlightening.")
+	medical_record_text = "Patient has acute phobia of light, and insists it is physically harmful."
+	hardcore_value = 4
+	mail_goodies = list(
+		/obj/item/flashlight/flashdark,
+		/obj/item/food/grown/mushroom/glowshroom/shadowshroom,
+		/obj/item/skillchip/light_remover,
+	)
+
+/datum/quirk/photophobia/add(client/client_source)
+	RegisterSignal(quirk_holder, COMSIG_CARBON_GAIN_ORGAN, PROC_REF(check_eyes))
+	RegisterSignal(quirk_holder, COMSIG_CARBON_LOSE_ORGAN, PROC_REF(restore_eyes))
+	RegisterSignal(quirk_holder, COMSIG_MOVABLE_MOVED, PROC_REF(on_holder_moved))
+	update_eyes(quirk_holder.get_organ_slot(ORGAN_SLOT_EYES))
+
+/datum/quirk/photophobia/remove()
+	UnregisterSignal(quirk_holder, COMSIG_CARBON_GAIN_ORGAN)
+	UnregisterSignal(quirk_holder, COMSIG_CARBON_LOSE_ORGAN)
+	UnregisterSignal(quirk_holder, COMSIG_MOVABLE_MOVED)
```

Comment:
Use `UnregisterSignals` instead

Reply:
How do I do that? Do I just add an 's' at the end of UnregisterSignal??

Reply:
I thought this was a thing but it wasn't, ignore me

Reply:
I'm thinking of this:

```dm
UnregisterSignal(quirk_holder, list(
    COMSIG_CARBON_GAIN_ORGAN,
    COMSIG_CARBON_LOSE_ORGAN,
    COMSIG_MOVABLE_MOVED,
))
```